vignettes/graph_gof_showcase.Rmd

```diff
@@ -76,17 +94,6 @@ tool:
          in this case they are in: /path/to/junit5/junit-platform-engine/build/classes/java/main/org/junit/platform/engine/
 ```
 
-```{r warning=FALSE,message=FALSE}
-require(kaiaulu)
-require(data.table)
-require(yaml)
-require(stringi)
-require(knitr)
-require(reticulate)
-require(magrittr)
-require(gt)
-```
-
 
```

Comment:
- After the configuration section, the notebook could be more explicit in stating what each command does. The output table is commented on, but does not really explain what the user is looking at. What does each row represent?
- The section on mapping to file paths could be separated, making clear that it is optional for users who want to see exactly where the patterns appear in the project. Should probably include what srcML is and how to install it.
- When you tried using this did it work for all projects, or only some? Testing it out on various projects should help with letting the user know how they can expect to use the tool. Be mindful that a process that takes you a long time could be near impossible for a user to navigate. The notebook should take the complex process, break it down, and make it easily digestible. The reason you should test on different projects is so the user does not have to. You can just tell them what works and what does not.